main.js

```diff
@@ -44,5 +44,20 @@ const toggleFooterMenu = (e) => {
 };
 
 toggleFooterMenu(mediaQueryList);
+mediaQueryList.addEventListener("change", toggleFooterMenu);
 
-mediaQueryList.addEventListener("change", toggleFooterMenu);
\ No newline at end of file
+let scrollButton = document.getElementById("scrollBtn");
+window.onscroll = function () { scrollFunction() };
+scrollButton.style.display = "none";
+
+function scrollFunction() {
+  if (document.body.scrollTop > 20 || document.documentElement.scrollTop > 20) {
```

Comment:
why exactly are you checking for both those values? DocumentElement gives you the html, and body - body. I guess you can stick just to one of them as one is a child of the other :)

Reply:
Have removed the body, when it was without the documentElement it wasn't working. But it works without the body